projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -283,7 +283,16 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
     if (0 === error.status || 502 === error.status) {
       return WriteDocumentFieldComponent.UPLOAD_ERROR_NOT_AVAILABLE;
     }
-    return error.error;
+
+    let errorMsg = null;
+    if (error && error.error) {
+      const fullError = error.error;
+      const start = fullError.indexOf('{');
```

Comment:
Need to check that the error message actually contains some json:
if (start >= 0) {